docs/tutorials/networks_seq2seq_nmt.ipynb

```diff
@@ -108,19 +108,6 @@
         "# **Setup**"
```

Comment:
<p>Could you move imports to <code>Additional Resources</code>  sections?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Can you expand on this? It seems that the imports are already in the `Additional Resources` section.

Reply:
Sure, I mean move
```python
from pickle import load
from numpy.random import shuffle
```
to this cell
```python
import string
import re
...
```

Reply:
Sure, I mean move
```python
from pickle import load
from numpy.random import shuffle
```
to this cell
```python
import string
import re
...
```